tests/test_fsx.py

```diff
@@ -298,6 +297,122 @@ def test_fsx_create_backup_without_copy_tags(self):
         expected_tags = [{'Key': 'test-tag', 'Value': 'backup-tag'}]
         self.assertEqual(expected_tags, backups['Backups'][0]['Tags'])
 
+    def test_fsx_delete_file_system_skip_snapshot(self):
```

Comment:
Feels odd to say, but the tests here feels a little extreme. I don't think we really need to get testing every parameter permutation on the api works. I think we can make do with just the test on `test_fsx_delete_file_system`